src/lib/widgets/now-playing/ui/swipe-down-bar.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import { detectSwipe } from '../lib/detect-swipe';
+
+  const dispatch = createEventDispatcher<{ minimize: void }>();
+
+  const { handlePointerMove, handlePointerStart, handlePointerUp } = detectSwipe({
+    down: () => {
+      dispatch('minimize');
+    },
+  });
+</script>
+
+<button
```

Comment:
question: why is it a button? Looks like this will produce very confusing screen reader semantics